docs/learn/protocols/iota2.0/core-concepts/consensus/preliminaries.md

```diff
@@ -0,0 +1,288 @@
+
+The actual moment depends on the [accepted Tangle time](tip-selection-algorithm.md#eligible-tips) of the node. More specifically, if one of the conditions holds:
+
+- the [accepted Tangle time](tip-selection-algorithm.md#eligible-tips) is larger than `slotEnd+minCommittableSlotAge*slotDurationSeconds`;
+- the current wall clock is at least `slotEnd+maxCommittableSlotAge*slotDurationSeconds`;
+
+Then the node generates the commitment of the required slot.
+
+### Reality-based UTXO ledger
+
+As [Chrysalis](../../../chrysalis/introduction.md) and [Stardust](../../../stardust/introduction.md), IOTA 2.0 uses the Unspent Transaction Output (UTXO) model for transactions. Its parallel processing capability allows independent transactions to be added to the ledger in any order, thus enhancing its scalability potential.
+
+However, the append-only nature of the UTXO ledger can compromise this benefit when conflicting transactions arise. To address this issue, IOTA 2.0 uses an enhanced UTXO ledger model, called the [reality-based UTXO ledger](preliminaries.md#reality-based-utxo-ledger), that optimistically updates the ledger and maintains a record of the dependencies of possible conflicts.
+
+A UTXO transaction should have a list of unique outputs called _UTXOs_. In addition, each transaction has a list of inputs, which are UTXOs of previous transactions. A transaction is said to spend its _inputs_.
+
+#### Ledger DAG
+
+The _ledger DAG_ is a DAG whose vertex set consists of all transactions. There is a directed edge between two transactions $tx$ and $ty$ if $tx$ spends the output of $ty$. The ledger DAG's root is the genesis with no outgoing edges.
+
+#### Causal history of a transaction
+
+The _causal history_ of a given transaction $tx$ consists of all transactions that can be reached from the transaction $tx$ by traversing the ledger DAG along the directed edges.
+
+#### Conflict
+
+A transaction $tx$ is a _conflict_ if there is another transaction $ty$ such that $tx$ and $ty$ attempt to spend at least one identical input.
+
+#### Conflicting transactions
+
+Two transactions $tx$ and $ty$ are called _conflicting_ if the causal history of $tx$ and the causal history of $ty$ contain some transaction $tx'$ and $ty'$ which spend at least one identical input.
+
+#### Non-conflicting transaction
+
+A transaction $tx$ is _non-conflicting_ if there is no transaction $ty$ such that $tx$ and $ty$ are conflicting.
+
+#### Rejected transaction
+
+A transaction $tx$ is _rejected_ if there is a transaction $ty$, which is conflicting with $tx$ and has been [accepted](consensus-flags.md#acceptance-of-conflicting-transactions).
+
+In other words, once a transaction gets [accepted](consensus-flags.md#acceptance-flag), all transactions conflicting with it get rejected. Rejected transactions are removed from the reality-based ledger.
+
+### Branches
```

Comment:
I would remove this header as what we have below is a part of a reality-based utxo ledger section.